packages/amplify-java-runtime-provider/src/utils/package.ts

```diff
@@ -0,0 +1,31 @@
+import childProcess from 'child_process';
+import path from 'path';
+import { PackageRequest, PackageResult } from 'amplify-function-plugin-interface/src';
+
+export async function packageResource(request: PackageRequest, context: any): Promise<PackageResult> {
+  const resourceDir = path.join(request.srcRoot);
+  const meta = context.amplify.getProjectDetails().amplifyMeta;
+  const zipFileName = Object.keys(meta.function)[0] + '-1.0-SNAPSHOT.jar';
```

Comment:
you should be zipping into request.dstFilename. Otherwise the platform won't know where to find the zip file